### PR TITLE
fix(GPayload): allowedMentions

### PR DESCRIPTION
**Pull request by Kill00**
**Please describe the changes this PR makes and why it should be merged:**

- [x] fix `allowedMention`

**Status and versioning classification:**

* Updated `GPayload.js`

### CODE REVIEW

**Comment by S222em** [comment]
I don't see how this would fix it.

**Comment by S222em** [comment]
```js
respond({ content: 'Hello', allowedMentions: { repliedUser: true, parse: ['users'] } });
 ```
 And added `console.log(this.data.allowed_mentions)`
 
 Console: `{ repliedUser: true, parse: [ 'users' ] }`
 
 This is without the changes you made. As I said, not a problem with GCommands.

**Comment by S222em** [comment]
@Kill00 I found the solution for you, use `replied_user` instead of `repliedUser`.

See https://discord.com/developers/docs/resources/channel#allowed-mentions-object

**Comment by Kill00** [comment]
`allowedMentions: { repliedUser: true, parse: ['users'] }` try without this

File: src/structures/GPayload.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ class GPayload {
             this.options.attachments = this.options.content instanceof MessageAttachment ? this.options.content : [];
         } else { this.data.content = this.options.content || null; }
 
-        this.data.allowed_mentions = this.options.allowedMentions ? this.options.allowedMentions : { parse: [], repliedUser: true };
+        this.data.allowed_mentions = this.options.allowedMentions ? this.options.allowedMentions : {};
```
Comment (S222em) [inline src/structures/GPayload.js:59]:
```js
this.data.allowed_mentions = this.options.allowedMentions ? { replied_user: this.options.allowedMentions.repliedUser, parse: this.options.allowedMentions.parse, roles: this.options.allowedMentions.roles, users: this.options.allowedMentions.users } : { parse: [], replied_user: true };
```

**Review by S222em — CHANGES_REQUESTED** [review]

**Comment by S222em** [comment]
Try it out now, does everything work?

**Comment by Kill00** [comment]
it works now thanks :)

**Review by S222em — APPROVED** [review]

**Pull request merged**